src/main/java/com/group6/harmoniq/models/UserRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.group6.harmoniq.models;
+
+import java.util.List;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface UserRepository extends JpaRepository<User, Integer> {
+    List<User> findBySpotifyId(String spotifyId);
+    List<User> findByUid(Long uid);
```

Comment:
```suggestion
    List<User> findByUid(long uid);
```

Reply:
Also since every user has a unique SpotifyId and Uid, the functions findBySpotifyId(String spotifyId); and findByUid (Long uid); should return type User, not type List<User>

Reply:
Good catch, will make this change before merging